public/static/locales/en/faq.json

```diff
@@ -123,7 +123,7 @@
       "question": "Whom should I trust?",
       "answer": [
         "In order to build a living, stable, basic income system with Circles, you should mainly trust people who bring value into the system. Ideally, you should trust people who will sell cupcakes or their bike for Circles, who will help their friends move and get paid for that with Circles, etc. If we have enough of this type of energy in the system, it will take off and fly, and once that happens, we will be able to better support those who truly need this kind of basic income, but are not in the position to give anything in return.",
-        "Don’t forget: Circles is about giving each other a basic income. We need here givers too."
+        "Don’t forget: Circles is about giving each other a basic income. We need benefactors too."
```

Comment:
Benefactors is perfectly fit! But I had to check the dictionary, and guess people would have to too... So not an issue imo.